src/transformers/modeling_flax_roberta.py

```diff
@@ -0,0 +1,49 @@
+import jax
+import numpy as np
+import jax.numpy as jnp
+from transformers import BertConfig
+
+from transformers import RobertaConfig
+from transformers.modeling_roberta import ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP
+from transformers.modeling_flax_bert import FlaxBertModel
+
+
+class FlaxRobertaModel(FlaxBertModel):
+    config_class = RobertaConfig
+    pretrained_model_archive_map = ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP
+    base_model_prefix = "roberta"
+
+    def __init__(self, config: BertConfig, state: dict, **kwargs):
+
+        super().__init__(config, state, **kwargs)
+
+        if config.pad_token_id is None:
+            config.pad_token_id = 1
+
+    @property
+    def config(self) -> RobertaConfig:
+        return self._config
+
+    def __call__(self, input_ids, token_type_ids=None, position_ids=None, attention_mask=None):
```

Comment:
How often is this called? RIght now it looks like you @jit the predict graph each time the function is called. Is that the intention? Can you define&compile the predict function elsewhere?